src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -152,85 +158,103 @@ def generate_MCP_1_default_config(self, filename):
+    cbsd_fad_records_iteration_0_sas_test_harness_1 = generateCbsdRecords([sas_test_harness_device_4,sas_test_harness_device_5],
                                                                           [[grant_request_2, grant_request_3], [grant_request_4]])
 
     # Generate Cbsd FAD Records for SAS Test Harness 0, iteration 1
-    cbsd_fad_records_sas_test_harness_0_iteration_1 = generateCbsdRecords([sas_test_harness_device_2,sas_test_harness_device_3],
+    cbsd_fad_records_iteration_1_sas_test_harness_0 = generateCbsdRecords([sas_test_harness_device_2,sas_test_harness_device_3],
                                                                           [[grant_request_1],[grant_request_2]])
 
     # Generate Cbsd FAD Records for SAS Test Harness 1, iteration 1
-    cbsd_fad_records_sas_test_harness_1_iteration_1 = generateCbsdRecords([sas_test_harness_device_6], [[grant_request_5, grant_request_6]])
+    cbsd_fad_records_iteration_1_sas_test_harness_1 = generateCbsdRecords([sas_test_harness_device_6], [[grant_request_5, grant_request_6]])
    
     # SAS Test Harnesses configuration  
     sas_test_harness_0_config = {
         'sasTestHarnessName': 'SAS-TH-1',
         'hostName': 'localhost',
         'port': 9001,
-        'serverCert': "certs/server.cert",
-        'serverKey': "certs/server.key",
-        'caCert': "certs/ca.cert"
+        'serverCert': os.path.join('certs', 'server.cert'),
+        'serverKey': os.path.join('certs', 'server.key'),
+        'caCert': os.path.join('certs', 'ca.cert')
     }
     sas_test_harness_1_config = {
         'sasTestHarnessName': 'SAS-TH-2',
         'hostName': 'localhost',
         'port': 9002,
-        'serverCert': "certs/server.cert",
-        'serverKey': "certs/server.key",
-        'caCert': "certs/ca.cert"
+        'serverCert': os.path.join('certs', 'server.cert'),
+        'serverKey': os.path.join('certs', 'server.key'),
+        'caCert': os.path.join('certs', 'ca.cert')
     }
 
     # Generate SAS Test Harnesses dump records for multiple iterations   
     sas_test_harness_0_dump_records_iteration_0 = {
-        'cbsdRecords': cbsd_fad_records_sas_test_harness_0_iteration_0
+        'cbsdRecords': cbsd_fad_records_iteration_0_sas_test_harness_0
     }
     sas_test_harness_0_dump_records_iteration_1 = {
-        'cbsdRecords': cbsd_fad_records_sas_test_harness_0_iteration_1
+        'cbsdRecords': cbsd_fad_records_iteration_1_sas_test_harness_0
     }
     sas_test_harness_1_dump_records_iteration_0 = {
-        'cbsdRecords': cbsd_fad_records_sas_test_harness_1_iteration_0
+        'cbsdRecords': cbsd_fad_records_iteration_0_sas_test_harness_1
     }
     sas_test_harness_1_dump_records_iteration_1 = {
-        'cbsdRecords': cbsd_fad_records_sas_test_harness_1_iteration_1
+        'cbsdRecords': cbsd_fad_records_iteration_1_sas_test_harness_1
     }
 
     # Create the actual config.
     iteration1_config = {
```

Comment:
In some places (e.g. `sas_test_harness_0_dump_records_iteration_0` and `sas_test_harness_0_dump_records_iteration_1`) you number the iterations as 0 and 1. In other places (`iteration1_config`, `cbsd_records_iteration_1_domain_proxy_0`) you number them as 1 and 2.

Please consistently zero-index.

Reply:
Unified   the naming conventions at appropriate places